samples/star/src/main/kotlin/com/slack/circuit/star/petlist/PetList.kt

```diff
@@ -140,7 +142,7 @@ object PetListScreen : Screen {
 
     data class NoAnimals(override val isRefreshing: Boolean) : State
     data class Success(
-      val animals: List<PetListAnimal>,
+      val animals: ImmutableList<PetListAnimal>,
```

Comment:
I assume the benefit here is that there is no need for a comparison when dealing with ImmutableList vs List during recomposition?

Reply:
Compose doesn't try `List` as stable or immutable, so it actually recomposes each time. ImmutableList gets special treatment